### ISSUE TITLE
FNAF 1 - CUSA16870

### ISSUE THREAD

**Issue by Boiiiiiiiiiiiiiiiiiiii** [red-prig/fpps4-game-compatibility#246]
Status: Nothing

Error: None

Black screen when in game menu

**Comment by gs-soda** [comment]
Update: Goes ingame, but as said, theres a black screen in the menu and in the game
red-prig/fpPS4@39a4e2c
Game version: 1.00
-h MEMORY_BOUND_HACK doesn't have to be used anymore

**Comment by gandalfthewhite19890404** [comment]
Add some screenshots, describe - are you sure game goes to menus and to actual gameplay?

**Comment by ArtemVideoGames** [comment]
> Add some screenshots, describe - are you sure game goes to menus and to actual gameplay?

I'm sure, I can send you screenshots

**Comment by ArtemVideoGames** [comment]
> Add some screenshots, describe - are you sure game goes to menus and to actual gameplay?

Menu:
![image](https://user-images.githubusercontent.com/122225808/214797060-036491d4-1004-4b52-9c65-34e50375ef55.png)

**Comment by AussiePlick** [comment]
Looks like black screen no visible menus.

**Comment by ArtemVideoGames** [comment]
> Looks like black screen no visible menus.

Yes, but all works. I played this game on laptop and i know all sounds in game

**Comment by ArtemVideoGames** [comment]
I think this game needs issues-video

**Comment by ArtemVideoGames** [comment]
Game goes to gameplay not only menu
